elasticjob-lite/elasticjob-lite-spring/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/spring/boot/tracing/ElasticJobTracingConfiguration.java

```diff
@@ -30,17 +34,40 @@
  */
 @EnableConfigurationProperties(TracingProperties.class)
 public class ElasticJobTracingConfiguration {
-    
+
+    /**
+     * Create a bean of tracing DataSource.
+     *
+     * @param tracingProperties tracing Properties
+     * @return tracing DataSource
+     */
+    @Bean("tracingDataSource")
+    public DataSource tracingDataSource(final TracingProperties tracingProperties) {
```

Comment:
maybe add a annotation here `@ConditionalOnProperty(name = "elasticjob.tracing.type", havingValue = "RDB")`

Reply:
现在使用 3.0.2 会由于这里初始化一个 DataSource 影响到了应用启动，因为没设置主要数据源等之类

Reply:
@cocodroid @TeslaCN 麻烦看一看。💗

Reply:
不用 tracing 的暂时如此处理
```java
public class RemoveElasticJobTracingBeanFactoryPostProcessor implements BeanDefinitionRegistryPostProcessor {

	public static final String ELASTIC_JOB_TRACING_CONFIGURATION_NAME
		= "org.apache.shardingsphere.elasticjob.lite.spring.boot.tracing.ElasticJobTracingConfiguration";

	@Override
	public void postProcessBeanDefinitionRegistry(@NonNull BeanDefinitionRegistry registry) throws BeansException {

		if (registry.containsBeanDefinition(ELASTIC_JOB_TRACING_CONFIGURATION_NAME)) {
			registry.removeBeanDefinition(ELASTIC_JOB_TRACING_CONFIGURATION_NAME);
		}
	}

	@Override
	public void postProcessBeanFactory(@NonNull ConfigurableListableBeanFactory beanFactory) throws BeansException {
		// NOOP
	}
}
```

Reply:
> @cocodroid @TeslaCN 麻烦看一看。💗

I will check it ,thanks.

Reply:
> 现在使用 3.0.2 会由于这里初始化一个 DataSource 影响到了应用启动，因为没设置主要数据源等之类

You can provide sample code configuration？ I don't think this is a problem.

Reply:
- Bean method 'sqlSessionTemplate' in 'MybatisAutoConfiguration' not loaded because @ConditionalOnSingleCandidate (types: javax.sql.DataSource; SearchStrategy: all) did not find a primary bean from beans 'xxxxxxDataSource', 'tracingDataSource'

![image](https://user-images.githubusercontent.com/16837364/204212325-c5024fc1-bb8e-46ac-bcca-9200c8b50ade.png)